comment_extractor.py

```diff
@@ -0,0 +1,49 @@
+from astroid.scoped_nodes import FunctionDef
+from astroid.scoped_nodes import ClassDef
+from astroid.manager import AstroidManager
+
+
+class DocumentedModule:
+    '''Encapsulates a module with all the classes and top level functions being
+    documented.'''
+
+    def __init__(self, module_path):
+        '''Takes a Python file located at module_path, reads the AST and
+        generates the appropriate documentation objects.
+        '''
+        ast = AstroidManager().ast_from_file(module_path)
```

Comment:
I think you can use `astroid.parse` here, which is the top-level API function (encouraged use).